tests/integration_tests/test_examples.py

```diff
@@ -23,7 +23,8 @@
     EXAMPLES_ROOT_DIR / 'torch/maml_ppo_half_cheetah_dir.py',
     EXAMPLES_ROOT_DIR / 'torch/maml_trpo_half_cheetah_dir.py',
     EXAMPLES_ROOT_DIR / 'torch/maml_vpg_half_cheetah_dir.py',
-    EXAMPLES_ROOT_DIR / 'torch/maml_trpo_ml10.py'
+    EXAMPLES_ROOT_DIR / 'torch/maml_trpo_ml10.py',
```

Comment:
i think your example tests should run automagically by virtue of being in `examples/` --- they only need to go here if they require special arguments

@gitanshu can explain